code/modules/mob/living/simple_animal/hostile/necro.dm

```diff
@@ -495,32 +481,18 @@
 /mob/living/simple_animal/hostile/necro/zombie/putrid/check_edibility(var/mob/living/carbon/human/target)
 	if(busy)
 		return 0
-	if(isjusthuman(target))
+	if(ishuman(target))
```

Comment:
same as the parent check edibility -- also wow it doesn't `return ..()` - not your fault but wow